### PR TITLE
Oll 264 creation of a new lecture table with his crud operations

### PR DESCRIPTION
**Pull request by twillsonepitech**
## Summary

Please write a summary of your changes. Provide any information that could be
useful to the reviewer.

## Checklist

**Bureaucracy**

- [ ] I set a descriptive title
- [ ] I set a proper description, providing enough information to the reviewer
- [ ] I considered squashing this PR, if needed
- [ ] I set the proper assignee
- [ ] I set the proper reviewers

**Quality Assurance**

- [ ] I tested the behavior of this PR locally and made sure it complies with
  the ticket description

### CODE REVIEW

File: packages/service-backend/src/lecture/lecture.dto.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+}
+
+export class UpdateLectureModel {
+  @ApiProperty({ description: 'Lesson ID' })
+  @IsUUID()
+  lessonId: string;
+
+  @ApiProperty({ description: 'Updated lecture data' })
+  @IsString()
+  data: string;
```
Comment (ThisisYoYoDev) [inline packages/service-backend/src/lecture/lecture.dto.ts]: `data` should be Optional

File: packages/service-backend/src/lecture/lecture.service.ts [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+            if (error instanceof Prisma.PrismaClientKnownRequestError) {
+                throw new ConflictException('Lecture already removed!');
+            }
+            throw new ConflictException('Lecture not deleted!');
+        }
+    }
+
+    async getLecture(lectureId: IdLectureModel): Promise<LectureModel> {
+        try {
+            const lectureDb = await prisma.lecture.findFirst({
```
Comment (ThisisYoYoDev) [inline packages/service-backend/src/lecture/lecture.service.ts]: `findFirst` should be `findOne`

**Review by ThisisYoYoDev — CHANGES_REQUESTED** [review]

File: packages/service-backend/src/lecture/lecture.dto.ts (hunk 1 above)
Comment (twillsonepitech) [reply packages/service-backend/src/lecture/lecture.dto.ts]: done

File: packages/service-backend/src/lecture/lecture.service.ts (hunk 2 above)
Comment (twillsonepitech) [reply packages/service-backend/src/lecture/lecture.service.ts]: findOne doesn't exist ?

Comment (ThisisYoYoDev) [reply packages/service-backend/src/lecture/lecture.service.ts]: `findUnique` sorry

Comment (twillsonepitech) [reply packages/service-backend/src/lecture/lecture.service.ts]: done

**Review by ThisisYoYoDev — APPROVED** [review]

**Pull request merged**